Description: configure.ac: add efl as internal dep for eio
Author: Stefan Schmidt <stefan@osg.samsung.com>
Origin: upstream
Applied-Upstream: 99c0c1cb620fa9de87b6efb39077c5507df4fbdb
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -4053,6 +4053,7 @@
 EFL_INTERNAL_DEPEND_PKG([EIO], [ecore])
 EFL_INTERNAL_DEPEND_PKG([EIO], [eet])
 EFL_INTERNAL_DEPEND_PKG([EIO], [eo])
+EFL_INTERNAL_DEPEND_PKG([EIO], [efl])
 EFL_INTERNAL_DEPEND_PKG([EIO], [eina])
 EFL_INTERNAL_DEPEND_PKG([EIO], [emile])
 
 EFL_ADD_LIBS([EIO], [-lm])
